pydash/numerical.py

```diff
@@ -383,6 +386,43 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, callback):
```

Comment:
Instead of `callback`, should probably be named `operator` or `operation` or `op`. The name "callback" is too generic while the function being passed in will only be a specific math operation.